package.json

```diff
@@ -14,6 +14,17 @@
     "devrun": "npm-run-all clean webpack-dev-server",
     "test": "echo 'Running JS tests' && jest"
   },
+  "jest": {
+    "transform": {
+      ".*": "./node_modules/babel-jest"
+    },
+    "moduleFileExtensions": [
+      "js",
+      "json",
+      "es6"
```

Comment:
I don't think we'll be using `.es6`, all our `.js` is gna be es6 because we're ultra-modern!